src/store/modules/target.js

```diff
@@ -994,6 +1046,50 @@ export default ({
         sender: serverNote.createdBy
       }
       return localNote
+    },
+    mapSubdomainsFromServerToLocal: (state, getters) => (subdomains) => {
+      const mappedSubdomains = []
+      subdomains.forEach(subdomain => {
+        const mappedSubdomain = getters.mapSingleSubdomainFromServerToLocal(subdomain)
+        mappedSubdomains.push(mappedSubdomain)
+      })
+      return mappedSubdomains
+    },
+    mapSingleSubdomainFromServerToLocal: (state) => (subdomain) => {
+      const createdDate = new Date(subdomain.createdAt)
+      const newSubDomain = {
+        id: subdomain.id,
+        name: subdomain.name,
+        added: createdDate.getFullYear() + '-' + (createdDate.getMonth() + 1) + '-' + createdDate.getDate(),
+        checking: false,
+        interesting: false,
+        vulnerable: false,
+        boubty: false,
+        ignore: false,
+        scope: false,
+        agent: [],
+        ipAddress: subdomain.ipAddress,
+        http: subdomain.hasHttpOpen,
+        isAlive: subdomain.isAlive,
+        ports: [],
+        services: [],
+        directories: []
+      }
+      return newSubDomain
+    },
+    mapSingleSubdomainFromLocalToServer: (state) => (subdomain) => {
+      const newSubDomain = {
+        name: subdomain.subDomainData.name,
+        target: subdomain.targetName,
+        rootDomain: subdomain.rootDomainName,
+        agent: [],
+        ipAddress: subdomain.subDomainData.ipAddress,
+        hasHttpOpen: subdomain.subDomainData.hasHttpOpen,
+        isAlive: subdomain.subDomainData.isAlive,
+        services: [],
+        directories: []
+      }
```

Comment:
we are missing the array of labels

https://github.com/reconness/reconness/blob/7fd8fa03ef76ae3c2e01323d843ffb96ddd78374/src/ReconNess.Entities/Subdomain.cs#L30

Reply:
this is not fixed yet